.travis.yml

```diff
@@ -78,6 +78,7 @@ jobs:
         use: firrtl_build
       script:
         - verilator --version
+        - sbt $SBT_ARGS +publishLocal
```

Comment:
So Treadle will use the `publishLocal`-ed FIRRTL classes from master, while chisel3 will still get the jar copied from the same FIRRTL version. This should all be fine, though it could maybe be streamlined later.